backend/server/repositories/bucket_repository.ts

```diff
@@ -1,11 +1,11 @@
-const Bucket = require('../models/bucket');
-const Promise = require('bluebird');
-const logger = require('../logger');
-const PageUtils = require('../utils/page_utils');
+import logger from '../logger';
+import { BucketModel } from '../models/bucket';
+import * as Promise from 'bluebird';
```

Comment:
kinda, but I dont like the list syntax. It makes it confusing and then allows people to get things out of order. Oh well, bluebird seems pretty good